### PR TITLE
State that modifying ConfigMaps triggers automatic application of new configs

### PR DESCRIPTION
**Pull request by rh-max**
In all relevant procedures in Monitoring.

### CODE REVIEW

**Comment by rh-max** [comment]
Hi @sichvoge , could you or someone from the team please check correctness of the descriptions of what happens upon modifying ConfigMaps in different scenarios? Thanks.

**Comment by sichvoge** [comment]
Seems you got a lgtm from @metalmatze. But I let him say something.

**Review by metalmatze — APPROVED** [review]
/lgtm

**Comment by rh-max** [comment]
@bergerhoffer , could you please review & merge this?

**Comment by rh-max** [comment]
@bergerhoffer , and please cherrypick into master.

File: modules/monitoring-configuring-a-persistent-volume-claim.adoc
Comment (rh-max) [reply modules/monitoring-configuring-a-persistent-volume-claim.adoc]: Thanks! Applied this, apart from the "pods" versus "Pods". As far as I can see, we have "pods" everywhere.

**Comment by rh-max** [comment]
@bergerhoffer This is for master, 4.1, and 4.2. Sorry for confusion. Do you need anything else before merging & cherry-picking? Thank you.

File: modules/monitoring-configuring-a-persistent-volume-claim.adoc
Comment (bergerhoffer) [reply modules/monitoring-configuring-a-persistent-volume-claim.adoc]: Yeah, the guidelines [1] say we should be using "Pod" uppercase, but honestly there is a lot of inconsistency since not everyone has been following that. So it's fine for now.

[1] https://github.com/openshift/openshift-docs/blob/enterprise-4.1/contributing_to_docs/term_glossary.adoc#pod

**Comment by bergerhoffer** [comment]
Thanks for the clarification @rh-max! Merging and cherry-picking!

**Pull request merged**
